api/modelmanager/modelmanager.go

```diff
@@ -530,3 +530,17 @@ func (c *Client) UnsetModelDefaults(cloud, region string, keys ...string) error
 	}
 	return result.OneError()
 }
+
+// ExportBundle exports the current model configuration.
+func (c *Client) ExportBundle(tag names.ModelTag) (params.StringResult, error) {
```

Comment:
It looks like we don't yet have a client for the Bundle facade in the JUju core codebase.
We'll need to create a new api/bundle package and add this method to that.